known-limitations.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Known Limitations
+summary:
+toc: false
+---
+
+This page describes limitations we've identified in the [CockroachDB 1.0](v1.0.html) release and plan to investigate and possibly resolve in future releases.
+
+<div id="toc"></div>
+
+## Removing all rows from large tables
+
+When removing all rows from a table via a [`TRUNCATE`](truncate.html) statement or a [`DELETE`](delete.html#delete-all-rows) statement without a `WHERE` clause, the entire operation is currently batched as a single [transaction](transactions.html). For large tables, this can cause the nodes containing the table data to either crash or exhibit poor performance due to elevated memory and CPU usage.
+
+As a workaround, when you need to remove all rows from a large table:
+
+1. Use [`SHOW CREATE TABLE`](show-create-table.html) to get the table schema.
+2. Use [`DROP TABLE`](drop-table.html) to remove the table.
+3. Use [`CREATE TABLE`](create-table.html) with the output from step 1 to recreate the table.
+
+## Schema changes within transactions
+
+Within a single [transaction](transactions.html):
+
+- DDL statements cannot follow DML statements. As a workaround, arrange DML statements before DDL statements, or split the statements into separate transactions.
+- A [`CREATE TABLE`](create-table.html) statement containing [`FOREIGN KEY`](foreign-key.html) or [`INTERLEAVE`](interleave-in-parent.html) clauses cannot be followed by statements that reference the new table.
+- A table cannot be dropped and then recreated with the same name. This is not possible within a single transaction because `DROP TABLE` does not immediately drop the name of the table. As a workaround, split the [`DROP TABLE`](drop-table.html) and [`CREATE TABLE`](create-table.html) statements into separate transactions.
+
+## Nodes need access to the first key-value range on startup
```

Comment:
@spencerkimball, @a-robinson, is this sufficient for documenting https://github.com/cockroachdb/cockroach/issues/13027? If not, please help me revise/expand as necessary.